examples/todos/src/actions/index.test.js

```diff
@@ -0,0 +1,25 @@
+import * as actions from './index'
+
+describe('todo actions', () => {
+  it('addTodo should create ADD_TODO action', () => {
```

Comment:
Normally we separate the method we want to test to one more nested describe, such as:
```javascript
describe('addTodo', () => {
  it('should create ....', () => {
  });
});
```